src/iotjs.c

```diff
@@ -128,6 +125,9 @@ static int iotjs_start(iotjs_environment_t* env) {
   const iotjs_jval_t process = iotjs_init_process_module();
   iotjs_jval_set_property_jval(global, "process", process);
 
+  // Release the global object
+  jerry_release_value(global);
```

Comment:
I guessed releasing `global` at the termination is right and safe. Can you kindly clarify my query?

Reply:
Global is not used in the code below. It is only needed for registering the "process" object. No need to hold the reference until the IoT.js terminates.

Reply:
As of my understanding, reference count for `global` is increased by `jerry_get_global_object` so calling `jerry_release_value` does not make the code unsafe. Succeeding `jerry_release_value` (-1) seems more natural for me as a pair of `jerry_get_global_object` (+1)

Reply:
@glistening yes, that's right.

Reply:
@LaszloLango Understood. We didn't need to hold `jglobal` at the very beginning. Thanks.

@glistening As wrote, my query wasn't why calling the release function as the pair. I know the reference counting internally works. I wondered what the thoughts of someone who wrote `jglobal`, which is released at the termination, was. It turns out unnecessary.

Reply:
@LaszloLango In `uv_loop_close`, isn't it possible to use `global` in the callback function?

Reply:
@hs0225 you can get the gobal object anytime before the jerry_cleanup just like before this patch.

Reply:
@LaszloLango I understand. There is no problem because `jerry_cleanup` is called later.